knotx-core/knotx-api/src/main/java/com/cognifide/knotx/api/CustomHandlebarsHelper.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Knot.x - Reactive microservice assembler - API
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.api;
```

Comment:
Just loud thinking, not saying anything is wrong here.
Is `knotx-api` good place for adding dependency to `com.github.jknack` and this interface?
Maybe we already have some modules naming twist when `api` module contains `communication models`.

How about those scenarios:
(A) Having this helper in `knotx-engine` where all stuff with handlebars is (cons is that dependency to `knotx-engine` is required while this module is rater not one we expect to be a dependency).
(B) Creating `knotx-events` (please suggest better name) which contains all communication models that are already in `knotx-api`. Then `knotx-api` can really consists of interfaces that may be implemented by external parties.